source/deimos/openssl/crypto.di

```diff
@@ -574,6 +574,42 @@ void OPENSSL_init();
  * non-zero. */
 int CRYPTO_memcmp(const(void)* a, const(void)* b, size_t len);
 
+static if (OPENSSL_VERSION_AT_LEAST(1, 1, 0, 0))
+{
+    /* Standard initialisation options */
+    enum OPENSSL_INIT_NO_LOAD_CRYPTO_STRINGS = 0x00000001L;
+    enum OPENSSL_INIT_LOAD_CRYPTO_STRINGS    = 0x00000002L;
+    enum OPENSSL_INIT_ADD_ALL_CIPHERS        = 0x00000004L;
+    enum OPENSSL_INIT_ADD_ALL_DIGESTS        = 0x00000008L;
+    enum OPENSSL_INIT_NO_ADD_ALL_CIPHERS     = 0x00000010L;
+    enum OPENSSL_INIT_NO_ADD_ALL_DIGESTS     = 0x00000020L;
+    enum OPENSSL_INIT_LOAD_CONFIG            = 0x00000040L;
+    enum OPENSSL_INIT_NO_LOAD_CONFIG         = 0x00000080L;
+    enum OPENSSL_INIT_ASYNC                  = 0x00000100L;
+    enum OPENSSL_INIT_ENGINE_RDRAND          = 0x00000200L;
+    enum OPENSSL_INIT_ENGINE_DYNAMIC         = 0x00000400L;
+    enum OPENSSL_INIT_ENGINE_OPENSSL         = 0x00000800L;
+    enum OPENSSL_INIT_ENGINE_CRYPTODEV       = 0x00001000L;
+    enum OPENSSL_INIT_ENGINE_CAPI            = 0x00002000L;
+    enum OPENSSL_INIT_ENGINE_PADLOCK         = 0x00004000L;
+    enum OPENSSL_INIT_ENGINE_AFALG           = 0x00008000L;
+    /* OPENSSL_INIT_ZLIB                         0x00010000L */
+    /* currently unused                          0x00020000L */
+    /* OPENSSL_INIT_BASE_ONLY                    0x00040000L */
+    /* OPENSSL_INIT flag range 0xfff00000 reserved for OPENSSL_init_ssl() */
+    /* Max OPENSSL_INIT flag value is 0x80000000 */
+
+    enum OPENSSL_INIT_ENGINE_ALL_BUILTIN =
+        (OPENSSL_INIT_ENGINE_RDRAND | OPENSSL_INIT_ENGINE_DYNAMIC
+        | OPENSSL_INIT_ENGINE_CRYPTODEV | OPENSSL_INIT_ENGINE_CAPI |
+        OPENSSL_INIT_ENGINE_PADLOCK);
+
+    void OPENSSL_cleanup();
+    int OPENSSL_init_crypto(ulong opts, const(OPENSSL_INIT_SETTINGS)* settings);
+    int OPENSSL_atexit(ExternC!(void function()) handler);
```

Comment:
Could we avoid the template instance here and use an alias ?

Reply:
I suppose we can, but why is an additional declaration better than a template instance?